src/wizard/boards.ts

```diff
@@ -2,23 +2,23 @@ import { html } from "lit";
 
 export const boardSelectOptions = html`
   <optgroup label="ESP32">
+    <option value="featheresp32">Adafruit ESP32 Feather</option>
     <option value="esp-wrover-kit">Generic ESP32 (WROVER Module)</option>
```

Comment:
Generic needs to stay on top. 

Let's just only sort the "Other" groups, the other ones are small and we sort them manually by popularity.

Reply:
Done. I've restored the sort in ESP32 and ESP8266 groups.